samples/app-auth/nodejs/dialogs/botSSOAuthDialog.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We don't need this file?
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/59052547/b67b2a6d-cbb2-46de-a887-b668964539e4)

Reply:
Yes not needed

Reply:
ok